app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java

```diff
@@ -30,9 +30,9 @@ public void onCreate() {
         Router.sharedRouter().setContext(getApplicationContext());
         Router.sharedRouter().map("/main", MainActivity.class);
         Router.sharedRouter().map("/main/:tab", MainActivity.class);
-        Router.sharedRouter().map("/map/:projectId", Map.class);
-        Router.sharedRouter().map("/projects/:projectId", Project.class);
-        Router.sharedRouter().map("/projects/:projectId/elements/:elementId", Editor.class);
+        Router.sharedRouter().map("/map/:projectId", Project.class);
```

Comment:
probably safer if we refactor this part now instead of later, so that Project.class lives on /projects, Page on /pages, etc